config/webpack/webpack.config.js

```diff
@@ -60,43 +66,29 @@ module.exports = (env = {}) => {
   return {
     bail: true,
     entry: [
-      LIB_PATHS.ABS.ENTRY,
+      path.resolve(ROOT, 'src/index.js'),
     ],
-    externals: {
-      react: {
-        root: 'React',
-        commonjs2: 'react',
-        commonjs: 'react',
-        amd: 'react'
-      },
-      'react-dom': {
-        root: 'ReactDOM',
-        commonjs2: 'react-dom',
-        commonjs: 'react-dom',
-        amd: 'react-dom'
-      },
-      'styled-components': {
-        amd: 'styled-components',
-        commonjs: 'styled-components',
-        commonjs2: 'styled-components',
-      },
-    },
```

Comment:
You added `webpack-node-externals` to `package.json` but you're not using it here.